os/arch/boardctl.c

```diff
@@ -71,6 +71,14 @@
  * Public Functions
  ****************************************************************************/
 
+#ifdef CONFIG_ARCH_BOARD_RTL8730E
+// TODO: Added this dummy API to fix build error. Remove after rtk provides implementation
+int board_reset(int x)
```

Comment:
Since we are adding this in boot.c let us remove this function from this file.

Reply:
Updated,
Thankyou